instrumentation/aws-lambda-1.0/library/README.md

```diff
@@ -6,6 +6,8 @@ This package contains libraries to help instrument AWS lambda functions in your
 To use the instrumentation, configure `OTEL_LAMBDA_HANDLER` env property to your lambda handler method in following format `package.ClassName::methodName`
 and use one of wrappers as your lambda `Handler`.
 
+In order to configure a span flush timeout (default is set to 1 second), please configure `OTEL_LAMBDA_FLUSH_TIMEOUT` env property. The value is in seconds.
```

Comment:
probably should follow standard config property naming? see #1414

Reply:
That's a very good idea, I'm all in favour of standardisation. This is of course a bit different than the "regular" usage of the config properties as here we're not configuring an instrumentation but a runtime wrapper (only via env property) that uses the aws lambda instrumentation

If such usage fits into the model that you've proposed (for me it sounds OK), I believe that the updated property name should be: `OTEL_INSTRUMENTATION_AWS-LAMBDA_FLUSH_TIMEOUT` There is also another config property, defining the handler, which would then become: `OTEL_INSTRUMENTATION_AWS-LAMBDA_HANDLER`.

@trask please have a look ^^

Reply:
that sounds good, with underscore instead of dash, e.g.

`OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT`
and
`OTEL_INSTRUMENTATION_AWS_LAMBDA_HANDLER`

Reply:
Ah yes, kebab-case is for dot-property names only. Silly me :) Updated PR, please have a look.